jobs/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: static
+title: Open positions
+excerpt: "We want You!"
+search_omit: true
+---
+<ul class="post-list">
+{% for post in site.categories.jobs %}
+  <li><article><a href="{{ site.url }}{{ post.url }}">{{ post.title }}</a></article></li>
+{% endfor %}
+</ul>
+
+## About us
+Upday is a start-up based in the heart of Europe’s most exciting city, Berlin. Used by millions of users in several European countries, upday is pre-installed on the -1 homescreen page of Samsung’s flagship devices.
```

Comment:
I suspect the "-1" is a term only know to us. It's probably enough to say just "the homescreen"